api/src/main/resources/workbench.yaml

```diff
@@ -680,6 +680,23 @@ paths:
           schema:
             $ref: "#/definitions/AuditBigQueryResponse"
 
+  /v1/cron/checkClusters:
+    get:
+      security: []
+      tags:
+        - offlineCluster
+        - cron
+      description: >
+        Endpoint meant to be called offline to trigger cluster checks and
```

Comment:
When do you plan on running this? Will that be a separate PR?

Seems like we should run it a couple times a day at least, so we don't wind up with clusters that never get upgraded since they are always in use at the time in question.

Reply:
Yes - follow-up, per PR description.

On the frequency, I'm not totally sure.

Either (1) run frequently, e.g. every 3h or (2) run this daily in the middle of the night (where people are least likely be to using notebooks)

Reply:
I think with the separate settings, running this multiple times a day sounds good.